submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/common/BaseIntegrationTest.kt

```diff
@@ -26,7 +26,7 @@ internal open class BaseIntegrationTest(private val tempFolder: TemporaryFolder)
 
     @BeforeAll
     fun beforeAll() {
-        if (System.getProperty("itest.mode") == "mongo") {
+        if (System.getProperty("testing.mode") == "mongo") {
```

Comment:
This property shouldn't be changed, the name is defined here:

https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/gradle.properties#L2

And it's referenced through several scripts and commands. Please revert this change.

Reply:
gradle propery is call `itest.mode` but gradle property and system properties are not the same if you check gradle tasks which transform this property into a system property you can notice is call `testing.mode`, do I am missing something?
```
val itest = tasks.create<Test>("itest") {
    dependsOn(copySqlSchema)
    testClassesDirs = sourceSets["itest"].output.classesDirs
    classpath = sourceSets["itest"].runtimeClasspath

    val testingMode = project.property("itest.mode")
    println("##### Running integration tests in $testingMode mode #######")
    systemProperty("testing.mode", testingMode)

    useJUnitPlatform()
    extensions.configure(JacocoTaskExtension::class) {
        setDestinationFile(file("$buildDir/jacoco/jacocoITest.exec"))
        classDumpDir = file("$buildDir/jacoco/classpathdumps")
    }
}
```

Reply:
Good catch man, then I think we should change to `itest.mode` [here](https://github.com/EBIBioStudies/biostudies-backend-services/blame/master/gradle/itest.gradle.kts#L24) instead of changing the `BaseIntegrationTest` class.

This typo was here from the very beginning, I'm afraid the tests were never running on mysql then 🤦‍♂️